racksdb/web/ui/package.json

```diff
@@ -13,7 +13,9 @@
     "format": "prettier --write src/"
   },
   "dependencies": {
-    "@heroicons/vue": "^2.0.18",
+    "@headlessui/vue": "^1.7.16",
```

Comment:
I don't see where you use tailwindcss/forms and headlessui/vue components in this PR, are you sure these dependencies are required for this change?

Reply:
No it was a mistake, these dependencies are not required for this change so i removed them.